jooby/src/main/java/org/jooby/internal/WebSocketImpl.java

```diff
@@ -143,6 +152,14 @@ public boolean isOpen() {
   }
 
   @Override
+  public void broadcast(final Object data, final SuccessCallback success, final OnError err)
+      throws Exception {
+    for (WebSocket ws : sessions) {
```

Comment:
The broadcast terminates on the first error. Wouldn't it be better to try to send to all endpoint, log the exception and throw a new one if there was at least one failure?